portafly/src/App.tsx

```diff
@@ -56,25 +18,16 @@ const PagesSwitch = () => (
   </SwitchWith404>
 )
 
-const Root = () => {
-  const history = useHistory()
-  const logoProps = React.useMemo(
-    () => ({
-      onClick: () => history.push('/')
-    }),
-    [history]
-  )
-  return (
-    <AppLayout
-      logo={Logo}
-      logoProps={logoProps}
-      navVariant="vertical"
-      navItems={navItems}
-      navGroupsStyle="expandable"
-    >
-      <PagesSwitch />
-    </AppLayout>
-  )
-}
+const App = () => (
+  <Router>
+    <LastLocationProvider>
+      <I18nProvider>
```

Comment:
In the future we can store the I18N settings either in cookies or bring them from ddbb and pass the locale here, no?

Reply:
Correct! (it's pointed out in the PR's description, in the snippet)